package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/ifactory-solutions/awesome-react-app.git"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "eslint . --no-eslintrc -c .eslintrc",
```

Comment:
`--no-eslintrc` is necessary so that it doesn't try to use `./hooks/eslint/.eslintrc.json`.